Nuro/Utilities/CoreData/CoreDataManager.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  CoreDataManager.swift
+//  Nuro
+//
+//  Created by Karen Natalia on 11/10/22.
+//
+
+import Foundation
+import CoreData
+
+class CoreDataManager {
+    
+    static let shared = CoreDataManager()
+    let container: NSPersistentContainer
+    let context: NSManagedObjectContext
+    
+    private init() {
+        container = NSPersistentContainer(name: "Nuro")
+        container.loadPersistentStores(completionHandler: { (_, error) in
+            if let error = error as NSError? {
+                fatalError("Unresolved error \(error), \(error.userInfo)")
+            }
+        })
+        context = container.viewContext
+    }
```

Comment:
I think there should be additional code to handle merge policies. Please check on this
```suggestion
        context = container.viewContext
         context.automaticallyMergesChangesFromParent = true
        context.mergePolicy = NSMergeByPropertyStoreTrumpMergePolicy
    }
```

Reply:
Yes, i have added `context.automaticallyMergesChangesFromParent = true` so that our data will always be up-to-date to the newest data from parent

Reply:
For mergePolicy, i have explored about it and i'm confused to choose between `.mergeByPropertyStoreTrumpMergePolicyType` or `.mergeByPropertyObjectTrumpMergePolicyType`

I'd like to discuss about this on monday 👍